### DISCUSSION TITLE
Setting AI Persona

### DISCUSSION THREAD

**Discussion by szarleydwarf** [anasfik/openai#118] [Q&A] [answered]
Hi.

My background is in iOS.
I created an app about two months ago just for iOS where I set chatGPT to answer as if he is Jesus.
I achieved this by adding content for the system/assistant in the URL request body.  

I am new to the Dart/Flutter world hence my confusion here.
I wanted to make this app a cross-platform one and started to learn and implement it with Flutter.
Is possible to do something similar with your package?

Kindest regards
Rad

**Comment by anasfik** [comment, marked as answer]
Hello,

Yes, this package is for you if you want to create a Flutter app that targets many platforms,. there is an active changes appliance that follows the OpenAI API's new changes, but the use case you just mentioned can be done easily, follow the docs, if you get stuck reach out.

Anas.

**Reply by szarleydwarf** [reply]
Hello Anas.
I appreciate you came back with an answer.
I've been playing around this for a week now.
Looks like I'm not bright enough as I could not figure this out yet, hence I've posted that question of how to add a body to that request that will specify how the assistant will respond.
Kindly,
Rad

**Reply by anasfik** [reply]
Hello,

This package abstracts everything to provide a developer-friendly interface for interacting with OpenAI APIs, for your use case you can start by upgrading the package to the latest version 4.1.5 (as of this time of writing), make sure to configure your API key, then literally copy the code snippet for [chat completion](https://github.com/anasfik/openai#create-chat-completion) and customize the messages you want for assistant, system, and user. set other fields you need such as the model name..., call the methods and you will get the response that you can use.

Anas.
